### ISSUE TITLE
Enable configurable set of artifacts

### ISSUE THREAD

**Issue by aashish24** [OpenImaging/miqa#384]

**Comment by curtislisle** [comment]
The KVis subcontractor, David Mackey, completed an initial implementation a few weeks back.  When the PR was reviewed, Zach had some style requests -- related to how to initialize the new Django models.  David tried, but was not able to implement all of Zach's suggestions successfully.  

The PR (https://github.com/OpenImaging/miqa/pull/574) has been updated to partially address the request. After about two weeks of engineering attempt, I directed Dave to continue with the needs for the Lung Cancer application, instead.

**Comment by zachmullen** [comment]
Thanks Curt. I can take over the branch if it's something we want to get merged.

**Comment by dzenanz** [comment]
Yes, we would like to have this feature.

**Comment by curtislisle** [comment]
@zachmullen , yes, please take a look.  David ran into some issues where the artifacts weren't initialized early enough in startup for the Vue UI to be responsive.  Maybe you can incorporate this more quickly.

**Comment by curtislisle** [comment]
An update, we have fallen back to the approach that worked for us and will revisit this issue when we have a consistent "annotation version" of MIQA.  We can then consider what parts of this solution we want to include through a PR.
